.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length = 160
```

Comment:
I thought we had a limit of 120 on our python services?

Reply:
I was going off [frontstage](https://github.com/ONSdigital/ras-frontstage/blob/ff22d18f7099fda9de91b9c7848b65912e090392/.flake8#L2).

Reply:
ah i was going off response ops https://github.com/ONSdigital/response-operations-ui/blob/e70b43db5ca1135b7bee7d16defbd4f507a5ec57/.flake8#L2 😆